api/app/main.py

```diff
@@ -86,8 +88,34 @@
 @app.get('/health')
 async def get_health():
     """ A simple endpoint for Openshift Healthchecks """
-    LOGGER.info('/health')
-    return {"message": "Healthy as ever"}
+    try:
+        parts = [
+            config.get('PATHFINDER_BASE_URI'),
+            '/apis/apps/v1beta1/namespaces/',
+            config.get('PROJECT_NAMESPACE'),
+            'statefulsets',
+            config.get('PATRONI_HEALTH_SUFFIX')
+        ]
+        url = reduce(os.path.join, parts)
```

Comment:
Man... I really miss template literals in Javascript

Reply:
Oh actually there is a similar way: https://realpython.com/python-string-formatting/#3-string-interpolation-f-strings-python-36

Reply:
What I'm really looking for, which os.path.join almost gives, is not caring about slashes. Often, in configuration, you'll forget to add a trailing slash - because you're not in the context of the code. So having a function that neatly takes all the parts and makes sure it joins them correctly is real sweet. So you can join ['a', 'b', 'c'] and it becomes 'a/b/c', similarly ['a/','b/', 'c/'] becomes 'a/b/c/' - doesn't throw in weird slashes. Problem with os.path.join, is that it's really made for folders, so if you give it ['b', 'c', '/d'] it gives you '/d' :(. I don't want to do a + b + c.

Reply:
I was referring to 
```
def greet(name, question):
...     return f"Hello, {name}! How's it {question}?"
```
but I see your point